docs/roles-and-permissions.md

```diff
@@ -0,0 +1,31 @@
+# Roles and Permissions 
+
+> WordPress has six pre-defined roles: Super Admin, Administrator, Editor, Author, Contributor and Subscriber. Each role is allowed to perform a set of tasks called Capabilities.
+
+These roles and their permissions can be explored in detail here: [https://wordpress.org/support/article/roles-and-capabilities/](https://wordpress.org/support/article/roles-and-capabilities/)
+
+Inside of the web stories app we are harnessing these role definitions to manage what users have permission to do. This document is meant to be a reference of what these roles mean for web stories specifically. 
+
+| Has permission? | Role |
+| --- | --- |
+| Can create new story | Admin, Editor, Author |
+| Can duplicate story XYZ | Admin |
```

Comment:
This currently does not have any permission check, but if anything, it would need a check for `edit_web-stories`